### PR TITLE
Add zuul tempest jobs

### PR DESCRIPTION
**Pull request by fao89**
https://logserver.rdoproject.org/27/427/92f3a196c16ad9deba0d9e1852496f914ab4ec89/github-check/heat-operator-tempest-multinode/0ad5969/controller/ci-framework-data/tests/test_operator/tempest-tests/stestr_results.html

### CODE REVIEW

**Comment by fao89** [comment]
Currently failing:
```
{2} heat_tempest_plugin.tests.functional.test_os_wait_condition.OSWaitCondition.test_create_stack_with_multi_signal_waitcondition [204.871425s] ... FAILED
{1} heat_tempest_plugin.tests.scenario.test_server_cfn_init.CfnInitIntegrationTest.test_server_cfn_init [21.303928s] ... FAILED
{0} heat_tempest_plugin.tests.functional.test_software_config.ParallelDeploymentsTest.test_deployments_metadata [139.077174s] ... FAILED
{3} heat_tempest_plugin.tests.scenario.test_aodh_alarm.AodhAlarmTest.test_alarm [1.185802s] ... FAILED
{0} heat_tempest_plugin.tests.scenario.test_base_resources.BasicResourcesTest.test_base_resources_integration [116.040087s] ... FAILED
{3} heat_tempest_plugin.tests.scenario.test_server_signal.ServerSignalIntegrationTest.test_server_signal_userdata_format_raw [1222.825997s] ... FAILED
{3} heat_tempest_plugin.tests.scenario.test_server_signal.ServerSignalIntegrationTest.test_server_signal_userdata_format_software_config [1224.424317s] ... FAILED
{3} heat_tempest_plugin.tests.scenario.test_server_software_config.SoftwareConfigIntegrationTest.test_server_software_config [1270.902584s] ... FAILED
```
https://logserver.rdoproject.org/27/427/9c5fdabebb936d0c3155b8ccc6677ed4f2ec00d0/github-check/heat-operator-tempest-multinode/72667a8/controller/ci-framework-data/tests/tempest/podman_tempest.log

**Comment by fao89** [comment]
recheck

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        - ^heat_tempest_plugin.*
+      cifmw_test_operator_tempest_exclude_list: |
+        heat_tempest_plugin.tests.functional.test_os_wait_condition.OSWaitCondition.test_create_stack_with_multi_signal_waitcondition
+        heat_tempest_plugin.tests.scenario.test_server_cfn_init.CfnInitIntegrationTest.test_server_cfn_init
+        heat_tempest_plugin.tests.functional.test_software_config.ParallelDeploymentsTest.test_deployments_metadata
+        heat_tempest_plugin.tests.scenario.test_aodh_alarm.AodhAlarmTest.test_alarm
+        heat_tempest_plugin.tests.scenario.test_base_resources.BasicResourcesTest.test_base_resources_integration
+        heat_tempest_plugin.tests.scenario.test_server_signal.ServerSignalIntegrationTest.test_server_signal_userdata_format_raw
+        heat_tempest_plugin.tests.scenario.test_server_signal.ServerSignalIntegrationTest.test_server_signal_userdata_format_software_config
+        heat_tempest_plugin.tests.scenario.test_server_software_config.SoftwareConfigIntegrationTest.test_server_software_config
```
Comment (fao89) [inline zuul.d/jobs.yaml:31]: these are currently failing, we can try enable them in follow up PRs

**Review by bshephar — APPROVED** [review]
Very nice!

**Comment by fao89** [comment]
OSPRH-7385

**Pull request merged**